libraries/engage/product/components/UnitQuantityPicker/UnitQuantityPicker.jsx

```diff
@@ -76,13 +97,18 @@ const UnitQuantityPicker = ({
   disabled,
   minValue,
   maxValue,
+  productProperties,
 }) => {
   const {
     buttonColor = colors.shade8,
     buttonBgColor = colors.primary,
     inputColor = colors.dark,
     inputBgColor = colors.shade8,
+    propertyLabel = '',
   } = useWidgetSettings('@shopgate/engage/product/components/UnitQuantityPicker') || {};
+
+  const property = productProperties.filter(({ label: l }) => l === propertyLabel)[0];
```

Comment:
this should not happen inside of the component, since it would be executed on every render. Please move it into a selector.

Also .find() would be better here instead of .filter()[0]